src/test/vstart_wrapper.sh

```diff
@@ -16,6 +16,7 @@
 # GNU Library Public License for more details.
 #
 
+CEPH_ROOT=${CEPH_ROOT:-..}
```

Comment:
i'd recommend fall back to `$(readlink -f $(dirname $0)/../..)`, what do you think?